### DISCUSSION TITLE
Removing unused entries

### DISCUSSION THREAD

**Discussion by NikitaPZ** [Vysp3r/ProtonPlus#247] [Q&A] [answered]
Is there any way to remove unused entries from the sidebar? For example, I have previously used the flatpak version of Steam and recently switched to the Arch repo package, but the flatpak entry is still there. I have flatpak currently removed on my system.

**Comment by Vysp3r** [comment, marked as answer]
That means you have some leftovers of your Steam Flatpak installation.

Simply run `rm -rf ~/.var/app/com.valvesoftware.Steam`

This will delete all the leftovers.

**Reply by Vysp3r** [reply]
Did that resolve your question?

**Reply by NikitaPZ** [reply]
Yeah, sorry I forgot to mark as answer
